**Fixed:**

* Lint error in v1 recipes when `run_exports` is a dict
